client/src/component/Navbar.js

```diff
@@ -1,8 +1,29 @@
 import React from 'react'
+import { Link, Outlet } from "react-router-dom";
```

Comment:
Im unsure where the prop outlet is being used or if this is just code that is built in now but will be implemented later.

Reply:
Yes, I will build this later.